### DISCUSSION TITLE
I have a column in a list called "Old Id", which I've mapped to managed property called "RefinableString100". Searching within Pnp modern search, does not return value, it is blank??  Have I done something wrong?

### DISCUSSION THREAD

**Discussion by Nehaghar** [microsoft-search/pnp-modern-search#2670] [Q&A] [answered]
RefinableString100 Text Multi Query -Retrieve Refine Sort Safe OWS_OLDID  OLDID
Old ID Number

From the list I am able to get other value except Old Id and same list column not able to fetch in filters. Could you please help me understand what am I doing is wrong

**Comment by Nehaghar** [comment]
Also in PNP filter webpart not able to get all the items or filter all the items. I have more than 100000 items in the list

**Reply by wobba** [reply]
You cannot retrieve more than 500 results per page I believe.

**Reply by Nehaghar** [reply]
Hi wobba,

It is so weird for date filters, Created dates working absolutely fine However when I am filtering for modified dates I am not able to fetch results. Please find attached images
<img width="703" alt="img1" src="https://user-images.githubusercontent.com/16334140/210712450-83db2de5-bd4e-4ba2-88eb-a0d567461a9c.png">
<img width="662" alt="img2" src="https://user-images.githubusercontent.com/16334140/210712498-a770a7c8-0aa8-4fed-974e-05c56e191e65.png">
<img width="682" alt="img3" src="https://user-images.githubusercontent.com/16334140/210712817-f4be1185-2012-45ce-a6c8-c28d777d2f20.png">
I have checked both the columns in the list (created date and modified date both has similar configuration). Plus, I am not getting any error. Not really sure what is the issue

**Comment by kasperbolarsen** [comment, marked as answer]
So a query like "RefinableString100:1*" does not return any hits?

At which level have you mapped the Refinablestring,  site collection or tenant?

**Comment by Nehaghar** [comment]
Thanks for reply @kasperbolarsen. It does but the user wanted to search just with the id so I have created new managed property and now able to search using the id. However, I am stuck with the filter issue not able to filter also the AND operator is not working. Some items I am able to fetch while some items I am not. Is it related to number of items I have in the list. Is there any limit for filtering items.
